website/src/components/Stats/Stats.components.tsx

```diff
@@ -38,30 +39,208 @@ type MessageTreeStateStatsProps = {
   };
 };
 
+const findLangRow = (rows: string[][], lang: string): number => {
+  for (let i = 0; i < rows.length; i++) {
+    const row_lang = rows[i][0];
+    if (row_lang === lang) return i;
+  }
+
+  return -1;
+};
```

Comment:
maybe use this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Thank you! `findIndex` was what I was looking for.